Readme.md

```diff
@@ -6,13 +6,21 @@ GGIRCS
 > for the Climate Action Secretariat
 
 ##Getting Started
```

Comment:
If you're adjusting documentation, there should be a space between the `##` and the `Getting` in Markdown